src/containers/ResourcesOverview/ResourcesOverview.jsx

```diff
@@ -60,7 +60,7 @@ const ResourcesOverview = ({ isAdmin }) => {
 
       {isAdmin ? (
         <>
-          <StaffResources cardsArray={resourceDataCopy} />
+          <StaffResources cardsArray={resourceData}/>
```

Comment:
line 11 can be removed and the mockData.resources import can be done directly on line 63. this is in the clients file too on line 10 and lines 41-44